### PR TITLE
check for redefined easyconfig parameters in easyconfig tests

### PR DESCRIPTION
**Pull request by boegel**
triggered by ~~#9875~~

There are couple more instances of redefined easyconfig parameters we'll need to fix... 

requires ~~#9875~~, ~~#9895~~ (fixes in OpenCV easyconfigs)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (11 easyconfigs in this PR)
node2430.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/25664380412dd4252ca90105bf78ac68 for a full test report.

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -820,6 +821,20 @@ def template_easyconfig_test(self, spec):
     error_msg = "Found use of '$root', not compatible with modules in Lua syntax, use '%%(installdir)s' instead: %s"
     self.assertFalse(res, error_msg % res)
 
+    # check for redefined easyconfig parameters, there should be none...
+    param_def_regex = re.compile('^(?P<key>[a-z_]+)\s*=', re.M)
```
Comment (smoors) [inline test/easyconfigs/easyconfigs.py]: @boegel some (local_*) parameters also contain digits and capital letters, so shouldn't this be:
```
param_def_regex = re.compile(r'^(?P<key>\w+)\s*=', re.M)
```

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 49 out of 75 (11 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f8e7cadc53f49192d5c4562270621081 for a full test report.

File: test/easyconfigs/easyconfigs.py (hunk 1 above)
Comment (boegel) [reply test/easyconfigs/easyconfigs.py]: Good point, fixed in 6daa654

**Comment by boegel** [comment]
@verdurin Installation of `OpenMPI-1.10.2-GCC-4.9.3-2.25.eb` required for `foss/2016a` failed with error below, which has nothing  to do with this PR. If you want to follow up on that, let's do so in a separate issue?

```
make[2]: Entering directory `/tmp/adamh/easybuild/build/OpenMPI/1.10.2/GCC-4.9.3-2.25/openmpi-1.10.2/ompi/mca/common/verbs_usnic'
  CC       libmca_common_verbs_usnic_la-common_verbs_usnic_fake.lo
common_verbs_usnic_fake.c:72:5: error: unknown field 'ops' specified in initializer
     .ops = {
     ^
```

**Comment by verdurin** [comment]
@verdurin agreed that's a separate problem.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**